sdk/core/azure-core/test/ut/transport_adapter_curl.cpp

```diff
@@ -3,39 +3,23 @@
 
 #include "transport_adapter_base.hpp"
 #include <azure/core/context.hpp>
+#include <azure/core/http/policy.hpp>
 #include <azure/core/response.hpp>
+
 #include <iostream>
 #include <string>
 #include <thread>
 
+// The next includes are from Azure Core private headers.
+// That's why the path starts from `sdk/core/azure-core/src/`
+// They are included to test the connection pool from the curl transport adapter implementation.
 #include <http/curl/curl_connection_pool_private.hpp>
```

Comment:
So, should these have `src` in the path? The comment says the path starts from `sdk/core/azure-core/src/`. How?
```C++
#include <src/http/curl/curl_connection_pool_private.hpp>
```

Reply:
No.
The trick is done in the CMake test project where we include that directory as part of the INCLUDE path:
```cmake
# Adding private headers from CORE to the tests so we can test the private APIs with no relative paths include.
target_include_directories (azure-core-test PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../src>)
```